roles/openshift_aws/tasks/iam_cert.yml

```diff
@@ -31,5 +33,5 @@
 
 - name: wait for cert to propagate
   pause:
-    seconds: 5
+    seconds: 10
```

Comment:
What's the logic behind the 10 seconds?  Would it be better to poll here?

Reply:
in my multiple rounds of testing, i saw a couple of cases where the 5 second wait wasn't long enough. after bumping it to 10, i stopped running into issues

Reply:
I hate indiscriminate pauses.  If this is a necessary evil then we can proceed for now but we need to figure out a way to poll and determine if this has arrived.

Maybe have a filter plugin that queries and returns an ARN for it.  Until then, loop for 1-2 min.  This should then give us the polling ability to determine if it has deployed.

I won't block on this but we should look to solve this.

Thoughts?

Reply:
i tried sticking an iam_server_certificate_facts and loop until we get an ARN, but it always returned for me on the first pass, so it was worse than then 5 second pause. we could loop on the ELB deletion though...